backend/de.metas.business/src/test/java/de/metas/document/dimension/InOutLineDimensionFactoryTest.java

```diff
@@ -46,7 +47,8 @@ void fieldsThatShallNotBeOverridden()
 		record.setC_Order_ID(9000001);
 		record.setM_Product_ID(9000002);
 
-		dimensionFactory.updateRecord(record, DimensionTest.newFullyPopulatedDimension());
+		final Dimension dimension = DimensionTest.newFullyPopulatedDimension();
+		dimensionFactory.updateRecord(record, dimension);
```

Comment:
why extracting it in a separate variable (`dimension`)?

Reply:
now, i think, u are also getting an IntelliJ warning

Reply:
It looked cleaner for me while debugging that code at some point. But I am changing it back since it doesn't seem to add value now.